RELEASE-NOTES.md

```diff
@@ -0,0 +1,16 @@
+# Nexus Repository R plugin
+
+## [v1.1.0](https://github.com/sonatype-nexus-community/nexus-repository-r/blob/nexus-repository-r-1.1.0/README.md)
+
+> **What's New* R is now part of Nexus Repository Manager. Version 3.20+ includes the R plugin by default.* The source code for the R plugin continues to live in this community project.
```

Comment:
It could be how I'm viewing it but the view of this looks weird.  If it's not just me, I suspect the *s are colliding.
I noticed this because "What's New R" looks weird in the preview.
If it's just me, feel free to ignore (I am viewing thru the github website, view file).

<img width="219" alt="Screen Shot 2019-12-02 at 5 55 19 PM" src="https://user-images.githubusercontent.com/9559205/70002125-1f3ca080-152d-11ea-8a21-dee014966513.png">

Reply:
38bbb2272a4ffb6f90784534ab4ba2c196991aef